From deaf18f77fc67d7be6534843a009a47e8df40eb4 Mon Sep 17 00:00:00 2001
From: Pino Toscano <toscano.pino@tiscali.it>
Date: Mon, 31 Jan 2022 21:29:37 +0100
Subject: [PATCH] CMake: enable geoclue2 also on GNU/Hurd

Geoclue builds also on GNU/Hurd (it is a D-Bus service), so build the
geoclue2 plugin there too.

Change-Id: If23edd7532a430e9df229eb1d293fb31090708fa
Reviewed-by: Alex Blasche <alexander.blasche@qt.io>
---
 src/plugins/position/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/position/CMakeLists.txt b/src/plugins/position/CMakeLists.txt
index ceb4fe27..9493a4ee 100644
--- a/src/plugins/position/CMakeLists.txt
+++ b/src/plugins/position/CMakeLists.txt
@@ -1,7 +1,7 @@
 # Generated from position.pro.
 
 add_subdirectory(positionpoll)
-if(FREEBSD OR LINUX OR OPENBSD OR (NETBSD AND TARGET Qt::DBus))
+if(FREEBSD OR LINUX OR OPENBSD OR (NETBSD AND TARGET Qt::DBus) OR HURD)
     add_subdirectory(geoclue2)
 endif()
 if(QT_FEATURE_gypsy)
-- 
2.35.1

